src/DependencyInjection/RollerworksPasswordStrengthExtension.php

```diff
@@ -70,4 +106,36 @@ private function configureChainBlacklistProvider(ContainerBuilder $container, ar
             $chainLoader->addMethodCall('addProvider', [new Reference($provider)]);
         }
     }
+
+    private function registerBlacklistCommands(ContainerBuilder $container, array $providers)
+    {
+        if (!class_exists(Application::class)) {
+            return;
+        }
+
+        $refs = ['default' => new Reference('rollerworks_password_strength.blacklist_provider')];
+        foreach ($providers as $name => $serviceId) {
+            $refs[$name] = new Reference('rollerworks_password_strength.blacklist.provider.'.$name);
+        }
+        $providersService = ServiceLocatorTagPass::register($container, $refs);
+
+        // To be replaced once https://github.com/symfony/symfony/pull/23834 is merged.
+        $r = $container->getReflectionClass(BlacklistListCommand::class);
+        $namespace = $r->getNamespaceName();
+
+        $finder = (new Finder())
+            ->in(dirname($r->getFileName()))
+            ->name('/.php$/');
+
+        /** @var SplFileInfo $file */
+        foreach ($finder as $file) {
+            $class = $namespace.'\\'.$file->getBasename('.php');
+
+            $container->addObjectResource($class);
```

Comment:
Why adding an object resource here ? We don't need to invalidate the container cache each time this file is edited. We only need to make the container track the directory

Reply:
These classes were autowired, I forgot to remove this piece of code :) good catch 👍